src/components/cards/CommentCard.tsx

```diff
@@ -65,7 +65,7 @@ export const CommentCard = ({
   const windowWidth = useWindowWidth()
   const isMobile = windowWidth < 600 && windowWidth !== 0
 
-  const [editedContent, setEditedContent] = useState('')
+  const [editedContent, setEditedContent] = useState((comment.details as { content: string }).content)
```

Comment:
```suggestion
  const content = (comment.details as { content: string }).content || ''
  const [editedContent, setEditedContent] = useState(content)
```

You don't have to do the same long type cast in three different places then.